applications/nrf_desktop/doc/hid_forward.rst

```diff
@@ -41,6 +41,10 @@ Complete the following steps to configure the module:
        The maximum number of supported HID reports (:kconfig:option:`CONFIG_BT_HOGP_REPORTS_MAX`) is set by default for the nRF Desktop dongle, which supports two peripherals with an average of six HID reports each.
        Make sure to align this configuration value for other use cases, for example, if the dongle supports more peripherals.
 
+#. The :ref:`CONFIG_DESKTOP_HID_FORWARD_ENABLE <config_desktop_app_options>` option selects :ref:`CONFIG_DESKTOP_HID_REPORTQ <config_desktop_app_options>` to automatically enable the :ref:`nrf_desktop_hid_reportq`.
+   The HID report queue utility is used to locally enqueue reports at the source to prevent HID report drops.
+   If needed, you can update the maximum number of enqueued HID reports (:ref:`CONFIG_DESKTOP_HID_REPORTQ_MAX_ENQUEUED_REPORTS <config_desktop_app_options>`).
+   See HID report queue utility documentation for details.
```

Comment:
add link :ref:`nrf_desktop_hid_reportq`

Reply:
Link is already added a few lines above. We do not duplicate links in similar situations

Reply:
Then remove it from there and place it here. Here you address user to see the documentation so we should enable user just to click it and not search few lines is that a correct link.